### DISCUSSION TITLE
Digit Preference Module

### DISCUSSION THREAD

**Discussion by jonthegeek** [OpenRBQM/gsm.digitpref#3] [Ideas]
Extracted from Discussion OpenRBQM/OpenRBQM.github.io#1

Digit Preference links with a few representative screenshots

![image](https://github.com/user-attachments/assets/6aceef5c-1ae6-496a-a858-241bb217d39c)
- Gilead ggplot prototype

![image](https://github.com/user-attachments/assets/bcad3490-86a6-4884-a0d3-299eb7e635a2)
- [nipnTK R Package Vignette](https://nutriverse.io/nipnTK/articles/dp.html)
- [JMP implementation](https://www.jmp.com/support/downloads/JMPC71_documentation/Content/JMPCUserGuide/OP_C_FR_0015.htm)
- [digitTests R Package](https://koenderks.github.io/digitTests/)
- [dataquier R Package](https://gitlab.com/libreumg/dataquier)
- [Visualization to detect risks of clinical data in Risk-Based Quality Management](https://www.pharmasug.org/proceedings/china2021/DV/Pharmasug-China-2021-DV030.pdf)

## Before the August Tech Team Meeting:

- Please review the links here. Does anything stand out to discuss at the August meeting?
- Please provide any other references or things you've done that you can share. If you can't post it here but would like to share, email Jon at jon.harmon1@gilead.com
- How would you use a Digit Preferences module? Should it be its own R package, or part of something larger? Would you use it in R scripts? As a Shiny module? Something else?

**Comment by dennison-hall** [comment]
### Digit preference wireframe

![Digit preference](https://github.com/user-attachments/assets/15bf48e5-59b2-411a-8b8b-89e68c57db30)
I am writing to share a set of prototypes with the community. These prototypes are conceptual in nature and are intended to serve as a foundation for sharing ideas and aligning on the direction of further development.

Please note that these prototypes do not imply any specific or existing implementation. They are purely illustrative and should be considered as a starting point for discussions and collaborative ideation.

**Comment by samussiah** [comment]
[Digit preference](https://github.com/RhoInc/sas-statistical-data-checks/blob/master/Macros/DigitPref.sas) developed with SAS and parameterized by subgroup and time interval.
